### PR TITLE
New package: TaskPlays.TSWModManager version 2.2.1

### PR DESCRIPTION
**Pull request by aaronliu0130**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118844)

### CODE REVIEW

**Comment by aaronliu0130** [comment]
uhh what is happening

**Comment by aaronliu0130** [comment]
Weird, even without switches the thing just installs without needing user input, it automatically goes into the progress bar. I'll search up some switches.

**Comment by stephengillie** [comment]
Hi @aaronliu0130,

Automatic Validation ended with: 
image

File: manifests/t/TaskPlays/TSWModManager/2.2.1/TaskPlays.TSWModManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: TaskPlays.TSWModManager
+PackageVersion: 2.2.1
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2022-09-01
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/t/TaskPlays/TSWModManager/2.2.1/TaskPlays.TSWModManager.installer.yaml:12]:
```suggestion
- Architecture: x64
```
`x86` validations are performed on a 32-bit OS.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**